backstop/tests/user-icon.html

```diff
@@ -22,6 +22,22 @@
       rel="stylesheet"
       href="../assets/demo.css"
     />
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/user"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/checkmark"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/away"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/close-small"
```

Comment:
I guess would be good to apply minification if that is the expected usage of those files... Its 1kb now, so mb not that critical.. Just now there is lag of icons appearance and of course, the more icons in the html, the more separate requests we do...

Reply:
minification where?

Reply:
in the js files with web components we download

Reply:
There is not much difference between minified and unminified. But updated url just in case